src/platforms/android/common/performance/included-instrumentation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Included Instrumentation
+sidebar_order: 21
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+<Note>
+
+This is only available since Sentry's Android SDK version `4.3.0`
+
+</Note>
+
+### Android's Activity Instrumentation
+
+The Activity's instrumentation, once enabled, captures transactions for each Activity's start up, Transaction names follow pattern `ActivityName`, for example for an Activity called `MainActivity`, The Transaction's name is `MainActivity` and its operation is `navigation`.
+
+The transaction starts before each Activity's `onCreate` method is called.
+
+_The Activity's instrumentation is enabled by default, but you may disable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.activity.enable" android:value="false" />
+</manifest>
+```
+
+The transaction finishes after each Activity's `onResume` method is executed.
+
+_The transaction finishes automatically, but you may disable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />
+</manifest>
+```
+
+You may finish the transaction manually based on your App's business logic, for example, the transaction should be only finished when your API call triggered on Activity's `onCreate` has been executed and you've shown the the data in the UI to the user, to do that you should do:
+
+```java
+import io.sentry.Sentry;
+import io.sentry.SpanStatus;
+
+ISpan span = Sentry.getSpan();
+if (span != null) {
+  span.finish(SpanStatus.OK);
+}
+```
+
+In case that the user left this Activity before you've finished the transaction manually, the transaction will be finished automatically once the `onDestroy` method is called.
```

Comment:
```suggestion
In case that the user left this Activity before you've finished the transaction manually, the SDK finishes the transaction automatically when the `onDestroy` method is called.
```